modules/context/context.go

```diff
@@ -222,6 +222,7 @@ func (ctx *Context) ServeContent(name string, r io.ReadSeeker, params ...interfa
 	ctx.Resp.Header().Set("Expires", "0")
 	ctx.Resp.Header().Set("Cache-Control", "must-revalidate")
 	ctx.Resp.Header().Set("Pragma", "public")
+	ctx.Header().Set("Access-Control-Expose-Headers", "Content-Disposition")
```

Comment:
Is there a difference between `ctx.Resp.Header` and `ctx.Header`?

Reply:
```suggestion
	ctx.Resp.Header().Set("Access-Control-Expose-Headers", "Content-Disposition")
```

Reply:
I don't think there is a difference, no. But for consistencies sake I'll change it.